practices/start_as_a_coder.md

```diff
@@ -0,0 +1,123 @@
+
+7. Tutorials fail to teach you the most important skill
+
+The most crucial skill problem-solving, which comes only through trial and error. Tutorials give you a highly refactored final version of the code, so you miss out on seeing the skill in action.
+
+8. Learn what NOT to learn
+
+Learning every is simply not humanely possible. First test out the waters by trying out everything, then choose the technology that you feel the best for you.
+
+9. Consistency
+
+Nobody became a pro by putting in 20 hours in a day, but a lot of people did it by putting in 1 hour for 20 days. Your brain needs time to digest the information, you feed it.
+
+10. Open Source can help you level up
+
+Open Source is a great way to learn by building amazing Softwares and tools. But be prepared to be constantly learning as you would be required to be constantly adapting to new technologies. 
+
+## Why You Might Become a Coder
+
+A good coder is not made in a day, a week or a month.
+It takes much grinding and hardwork to polish your skills. These tips are just to give you a roadmap and help you out in your coding journey.
+
+## Problems The Coder Solves and problems that these tips would solve
+
+As a coder you would-
+You can solve any problem from searching, sorting or manipulating to automating, analysis and development.
+
+these tips would help you-
+for beginners to start a smoother journey.
+for the experienced ones to become more efficient and have trust in themselves.
+
+These tips might seem very normal and basic but are missed out by most of us so consider these all just to remind you of the best practices you should follow.
+
+
+## How to Become a Coder?
+
+Just follow these advices and suggestions daily throughout your coding journey.
+Or feel free to come back and read them again to build more confidence.
+They could prove to be very motivating and guide you to the right path.
+
+## Common Pitfalls of the Path to Become a Coder
+
+**Making it seem too easy**
+This article might make you look at things lightly which is not actually true. Coding requires consistent efforts and practice.
+
+**May only follow half the tips**
+There are so many tips and advices that you can feel overwhelmed. But do not worry - once you make youself comfortable with coding, you will realize that they are here to make your life easier, not more cumbersome.
+
+**These are not just all**
+These are only few of the many tips you can get. But I felt they were the most important so added these in this article.
+
+## Resources to Become a Coder
+
+**Find more tips**
+
+- Medium: [To get a whole bunch of tips you can refer to this article](https://medium.com/@tapajyoti-bose/25-tips-i-wish-i-knew-before-i-started-to-code-2c6c1af20b71)
+- Analytics Insight: [Tips to make your programming joourney easier](https://www.analyticsinsight.net/top-10-secret-coding-tips-to-make-your-programming-journey-easier/)
```

Comment:
Please update the link according to the title of the external article for the proper attribution of the source.